tools/formatter/format.mk

```diff
@@ -72,6 +72,10 @@ lint/line-endings:
 	else echo all files have unix line endings; \
 	fi
 
+.PHONY: lint/markdown-links
+lint/markdown-links:
```

Comment:
Just a note. This one `lint/markdown-links` will not be called if using `docker/format`.

Reply:
I know. Same as with all other `lint` make targets (and some of then are not covered by the `format` target, just like this one).
I have few ideas on how to improve it, but I'm yet to formalize them to create an issue for it.

Reply:
Ok. Because I see you added the tool to `Dockerfile` without using it, so just a note.

Reply:
Yeah, I launched the docker manually for this